apps/main/src/dao/store/createUserSlice.ts

```diff
@@ -106,7 +106,7 @@ const createUserSlice = (set: SetState<State>, get: GetState<State>): UserSlice
   [sliceKey]: {
     ...DEFAULT_STATE,
     updateUserData: async (curve: CurveApi, wallet: Wallet) => {
-      const userAddress = getWalletSignerAddress(wallet)!
+      const userAddress = wallet.account.address!
```

Comment:
Assert shouldn't be needed anymore
```suggestion
      const userAddress = wallet.account.address
```